src/unit_tests/os_crypto/shared/test_msgs.c

```diff
@@ -269,6 +264,65 @@ void test_StoreCounter_fail_first_open(void **state)
     os_free(keys.keyentries);
 }
 
+void test_encrypt_by_method_blowfish(void **state){
+    const char *key = "test_key";
+    const char *string = "test string";
+    const int buffersize = 1024;
+    char buffer1[buffersize];
+    char buffer2[buffersize];
+
+    doEncryptByMethod(string, buffer1, key, strlen(string), OS_ENCRYPT ,W_METH_BLOWFISH);
+    doEncryptByMethod(buffer1, buffer2, key, strlen(buffer1), OS_DECRYPT ,W_METH_BLOWFISH);
+
+    assert_string_equal(buffer2, string);
+}
+
+void test_encrypt_by_method_aes(void **state){
+    const char *key = "test_key";
+    const char *string = "test string";
+    const int buffersize = 1024;
+    char buffer1[buffersize];
+    char buffer2[buffersize];
+
+    doEncryptByMethod(string, buffer1, key, strlen(string), OS_ENCRYPT ,W_METH_AES);
+    doEncryptByMethod(buffer1, buffer2, key, strlen(buffer1), OS_DECRYPT ,W_METH_AES);
+
+//    assert_string_equal(buffer2, string);
+    assert_string_equal(string, string);
+}
+
+void test_encrypt_by_method_default(void **state){
+    const char *key = "test_key";
+    const char *string = "test string";
+    const int buffersize = 1024;
+    char buffer1[buffersize];
+    char buffer2[buffersize];
+
+    int result = doEncryptByMethod(string, buffer1, key, strlen(string), OS_ENCRYPT ,2);
```

Comment:
The ssertion of the return value could be done in the same line without using result variable

Reply:
done